app/src/main/res/values/strings.xml

```diff
@@ -99,10 +99,13 @@
     <string name="auto_queue_title">Auto-enqueue next stream</string>
     <string name="auto_queue_summary">Continue ending (non-repeating) playback queue by appending a related stream</string>
     <string name="auto_queue_toggle">Auto-enqueuing</string>
-    <string name="volume_gesture_control_title">Volume gesture control</string>
-    <string name="volume_gesture_control_summary">Use gestures to control player volume</string>
-    <string name="brightness_gesture_control_title">Brightness gesture control</string>
-    <string name="brightness_gesture_control_summary">Use gestures to control player brightness</string>
+    <string name="left_gesture_control_summary">Change the left gesture control</string>
+    <string name="left_gesture_control_title">Left gesture control</string>
+    <string name="right_gesture_control_summary">Change the right gesture control</string>
+    <string name="right_gesture_control_title">Right gesture control</string>
```

Comment:
I mainly meant from the user perspective:

> I feel like "left/right gesture control" is not easy to understand. What about "Action for gesture on left/right side"?

```suggestion
    <string name="left_gesture_control_summary">Choose the action for gestures on the left side of the player</string>
    <string name="left_gesture_control_title">Left gesture action</string>
    <string name="right_gesture_control_summary">Choose the action for gestures on the right side of the player</string>
    <string name="right_gesture_control_title">Right gesture action</string>
```

@opusforlife2 do you have any suggestion?

Reply:
@opusforlife2 I used your suggestion for the summarys. @Stypox I used your suggestion for the titles.